content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,74 @@
+want.
 
-.. image:: media/lead_scoring02.png
+.. image:: lead_scoring/scoring-example.png
    :align: center
+   :alt: Score Rule example
 
-Every hour every lead without a score will be automatically scanned and
-assigned their right score according to your scoring rules.
+Every hour, Odoo automatically assigns scores to new leads according to your configured rules.
 
-.. image:: media/lead_scoring03.png
+.. image:: lead_scoring/scoring-section-on-lead.png
    :align: center
+   :alt: Scoring section on a lead
 
 Assign leads
 ============
 
-Once the scores computed, leads can be assigned to specific teams using
-the same domain mechanism. To do so go to :menuselection:`CRM --> Leads Management --> Team Assignation`
-and apply a specific domain on each team. This domain can include scores.
+Odoo can also assign leads to specific teams based on configured rules. To configure assignment
+rules, go to the individual Sales Team's configuration page via :menuselection:`CRM --> Leads -->
+Team Assignment` or :menuselection:`CRM --> Configuration --> Sales Teams`, and apply a specific
+Domain to each team. The Domain can include lead scores.
 
-.. image:: media/lead_scoring04.png
+.. image:: lead_scoring/team-assignation.png
    :align: center
+   :alt: Team Assignments using domains
 
-Further on, you can assign to a specific vendor in the team with an even
-more refined domain.
+Assign leads to specific salespeople using further refined Domains at :menuselection:`CRM --> Leads
+--> Leads Assignment`.
 
-To do so go to :menuselection:`CRM --> Leads Management --> Leads Assignation`.
-
-.. image:: media/lead_scoring05.png
+.. image:: lead_scoring/lead-assignment-filters.png
    :align: center
+   :alt: Lead Assignments
 
-.. note::
-   The team & leads assignation will assign the unassigned leads
-   once a day.
+Odoo automatically assigns new leads to teams/salespeople once per day.
 
 Evaluate & use the unassigned leads
 ===================================
 
-Once your scoring rules are in place you will most likely still have
-some unassigned leads. Some of them could still lead to an opportunity
-so it is useful to do something with them.
+Not all leads will fall into your set Domains and get assigned automatically by Odoo.
 
-In your leads page you can place a filter to find your unassigned leads.
+To view your unassigned leads, go to :menuselection:`CRM --> Leads --> Leads` and select the
+Unassigned filter.
 
-.. image:: media/lead_scoring06.png
+.. image:: lead_scoring/unassigned-filter.png
    :align: center
+   :alt: Filtering for unassigned leads
 
-Why not using :menuselection:`Email Marketing` or
-:menuselection:`Marketing Automation` apps to send a mass email to
-them? You can also easily find such unassigned leads from there.
-
-.. image:: media/lead_scoring07.png
-   :align: center
+.. note::
+   You can also find unassigned leads using the **Email Marketing** or **Marketing
+   Automation** apps to create a re-engagement campaign.
```

Comment:
These should use `:guilabel:`, and I've rearranged the wording a bit for clarity.
```suggestion
   You can also find unassigned leads and create re-engagement campaigns using the :guilabel:`Email Marketing` or :guilabel:`Marketing
   Automation` apps.
```